searx/engines/reuters.py

```diff
@@ -84,7 +84,7 @@ def response(resp) -> EngineResults:
                 content=result["description"],
                 thumbnail=result.get("thumbnail", {}).get("url", ""),
                 metadata=result.get("kicker", {}).get("name"),
-                publishedDate=datetime.strptime(result["display_time"], "%Y-%m-%dT%H:%M:%SZ"),
+                publishedDate=datetime.fromisoformat(result["display_time"]),
```

Comment:
After this change, the result date sometimes looks like `2021-05-12 08:10:47.323000+00:00`. I think we should remove the microseconds and time zone info.
```suggestion

publishedDate=datetime.fromisoformat(result["display_time"]).replace(tzinfo=None, microsecond=0),
```

Or possibly that should be done for all templates in a preprocessing step before rendering the results, opinion @return42 ?

Reply:
Maybe the format used for rendering the datetime should not include TZ and microseconds (since it seems like a UI/UX thing)

Reply:
> Maybe the format used for rendering the datetime should not include TZ and microseconds (since it seems like a UI/UX thing)
>
Yeah, I agree. Perhaps we should open a new issue about that.

Reply:
> Maybe the format used for rendering the datetime should not include TZ and microseconds (since it seems like a UI/UX thing)

@ishbir please open a new issue .. so we have it on the list :+1:

Reply:
I've opened an issue.